library/src/test/java/io/constructor/core/ConstructorIoIntegrationTest.kt

```diff
@@ -99,6 +111,19 @@ class ConstructorIoIntegrationTest {
         Thread.sleep(timeBetweenTests)
     }
 
+    @Test
+    fun getAutocompleteResultsCRTWithSectionFiltersAgainstRealResponse() {
+        runBlocking {
+            val sectionFilters = mapOf(
+                "Products" to listOf("Brand" to listOf("XYZ")),
+            )
+            val autocompleteResults = constructorIo.getAutocompleteResultsCRT("item", sectionFacets = sectionFilters)
+            assertTrue(autocompleteResults.sections!!.isNotEmpty())
+            assertTrue(autocompleteResults.resultId!!.isNotEmpty())
```

Comment:
Could we add an assertion here to check that Product results are returned (when section filters are applied)?